### DISCUSSION TITLE
Keuze oplossing RFC####-Onderwerp

### DISCUSSION THREAD

**Discussion by rvanrest** [iStandaarden/iWlz_RequestForChange#17] [Enquete]
Het betreft de oplossingsrichting zoals beschreven in **#16 RFC####-Onderwerp**

**Comment by IZijlstra** [comment]
ik ben het er niet mee eens

**Reply by RTrigg1** [reply]
Ik wel

**Reply by vanrest-remo** [reply]
Zo de stand is weer gelijk! 🥇
